README.md

```diff
@@ -84,7 +86,19 @@ We recommend you use IntelliJ as your IDE. The code style template for the proje
 * Use an assertion from Airlift's `Assertions` class if there is one that covers your case rather than writing the assertion by hand. Over time we may move over to more fluent assertions like AssertJ.
 * When writing a Git commit message, follow these [guidelines](https://chris.beams.io/posts/git-commit/).
 
-## Building the Web UI
+### Additional IDE configuration
+
+When using IntelliJ to develop Presto, we recommend the following inspections to be turned on:
+
+- all inspections that come enabled by default except these listed below,
```

Comment:
I think you mean to say

```
- all inspections that come enabled by default, plus the following:
```

The current wording seems to say that those should be disabled.

Reply:
I see what you mean. I wrote so to signal there is _"We recommend the following inspections to be turned off"_ below.
So I can't really say that "all inspections that come enabled by default should be turned on", right?

Reply:
How about something like

> We recommend starting with all of the default inspections, with some modifications.
> 
> Enable these:
>
> Disable these:

Reply:
Good idea, applied